oscal.yaml

```diff
@@ -0,0 +1,98 @@
+# add the descriptions inline
+component-definition:
+  uuid: fae03d9e-4cfc-47ca-b0e9-05dae10c557b
+  metadata:
+    title: Podinfo
+    last-modified: "2024-06-14T12:00:00Z"
+    version: "20240614"
+    oscal-version: 1.1.2
+    parties:
+      # Should be consistent across all of the packages, but where is ground truth?
+      - uuid: c18f4a9f-a402-415b-8d13-b51739d689ff
+        type: organization
+        name: Defense Unicorns
+        links:
+          - href: https://github.com/defenseunicorns
+            rel: website
+  components:
+    - uuid: a9d5204c-7e5b-4c43-bd49-34df759b9f04
+      type: software
+      title: Podinfo
+      description: |
+        Defense Unicorns UDS Common Podinfo
+      purpose: Common tasks for UDS Packages
+      responsible-roles:
+        - role-id: provider
+          party-uuids:
+            - c18f4a9f-a402-415b-8d13-b51739d689ff # matches parties entry for Defense Unicorns
+      control-implementations:
+        - uuid: d2afb4c4-2cd8-5305-a6cc-d1bc7b388d0c
+          source: https://raw.githubusercontent.com/GSA/fedramp-automation/93ca0e20ff5e54fc04140613476fba80f08e3c7d/dist/content/rev5/baselines/json/FedRAMP_rev5_HIGH-baseline-resolved-profile_catalog.json
+          description: Validate generic security requirements
+          implemented-requirements:
+            - uuid: 7faf6742-a23d-41c1-9ba7-2072494aedc2
```

Comment:
```suggestion
            # TODO: (@WSTARR) - make this more relevant once this podinfo package is integrated further with UDS Core (i.e. authservice)
            - uuid: 7faf6742-a23d-41c1-9ba7-2072494aedc2
```

Reply:
Just adding a future note here - would be more cool if this checked authservice was in front of podinfo and providing auth through keycloak

Reply:
Added!

We can for sure throw up checks like that as more is added. I think it'll give us more to do stuff with on the OSCAL side in the future too.